index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Theophilus Doku</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <!-- Header -->
+    <header class="header-wrapper">
+        <a href="#" class="logo">
+            <h1>Sozo</h1>
+        </a>
+        <button class="menu_btn" type="button">
+            <img src="images/menu.png" alt="menu-button">
+        </button>
+    </header>
+
+    <!-- Headline Section -->
+    <section class="headline">
+        <div class="container">
+            <div class="title">
+                <h2>
+                    I'm Theo <br>
+                    Glad to see you!
+                </h2>
+            </div>
+            <div class="info">
+                <p>
+                    I’m a software developer! I can help you build a product , feature
+                    or website Look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                </p>
+            </div>
+            <div class="social-media">
+                <a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] This acts as a heading for the social media icons, I think an `<h2>` tag would fit best in this case. Please consider changing it! 🤓 🛠️ 

```suggestion
                <h2>LET’S CONNECT</h2>
```